flow/layers/physical_shape_layer.h

```diff
@@ -55,6 +55,10 @@ class PhysicalShapeLayer : public ContainerLayer {
 
 #if defined(OS_FUCHSIA)
   std::unique_ptr<scenic::EntityNode> entity_node_ptr_;
+
+  // Whether the Fuchsia scenic::EntityNode of this exact same layer has already
+  // been painted in previous frames. If so, we can skip the Paint phase and
```

Comment:
has been painted in a previous frame. This means that the content below the EntityNode has already been rasterized.

"and let Scenic re-use the same Entity nodes."

Reply:
Done.